packages/itwinui-css/src/button/base.scss

```diff
@@ -50,7 +50,7 @@
   --_iui-button-active-stripe-inset-inline: initial;
   --_iui-button-active-stripe-background: var(--iui-color-border-accent);
   --_iui-button-text-color: var(--iui-color-text-accent);
-  --_iui-button-icon-fill: var(--iui-color-text-accent);
+  --_iui-button-icon-fill: var(--iui-color-icon-accent);
```

Comment:
i feel like we had a specific reason we used text color here 🤔 or maybe I'm misremembering

Reply:
I think I remember that too.  I think we thought the icon next to the text looked off.

**With this change:**
![Screenshot 2024-01-18 at 6 16 12 PM](https://github.com/iTwin/iTwinUI/assets/849817/6bc596c4-ef63-424b-8726-4791bfc2a843)

I think we should use the icon color now to avoid having to explain why we're using text for icon fill.